### ISSUE TITLE
Radio field, list and checkbox produce a single value rather than an array

### ISSUE THREAD

**Issue by johardi** [metadatacenter/cedar-template-editor#627]
The value schema definition for those fields says that they can only be an array or a null value.
```
"@value": {
    "type": [
        "array",
        "null"
    ]
}
```
However, their value in the instance is presented as a single string.

**Comment by martinjoconnor** [comment]
@marcosmro can comment, but I think we decided that radio buttons (which have only once final value in the instance) use single values.

**Comment by marcosmro** [comment]
That's right. Radio buttons and single-choice list fields generate single values. Checkboxes and multi-choice list fields generate an array of values.

**Comment by martinjoconnor** [comment]
Changing this from a bug to a model issue. Model schema should allow both variants.

**Comment by martinjoconnor** [comment]
Merged into https://github.com/metadatacenter/cedar-model-validation-library/issues/31

**Comment by johardi** [comment]
The value type for the list field, checkbox field, and radio field is still an array. The correct type should be either a "string" or "null".

**Comment by johardi** [comment]
In addition the "type" specification for the list field and checkbox field should be an "array" instead of "object"

**Comment by marcosmro** [comment]
I've set the value type for checkbox, radio, single-select list, and multi-select list to either a "string" or "null". Fixing the type specification of checkboxes and multi-select lists to be an "array" instead of "object" is a more complex task that will be addressed by https://github.com/metadatacenter/cedar-model-validation-library/issues/31

**Comment by martinjoconnor** [comment]
Verifying that this now allows #22 (complete artifact validation/fix script) to go forward.
